test/routing.spec.js

```diff
@@ -30,7 +31,7 @@ describe('RoutingTable', () => {
   })
 
   it('add', (done) => {
-    createPeers(20, (err, peers) => {
+    createPeers(10, (err, peers) => {
```

Comment:
Please, do not change the tests

Reply:
timeout issues on Travis CI

Reply:
You can always increase the timeout

Reply:
The timeout is already at 10 seconds.  Unit tests are suppose to be quick.